pkg/kubelet/prober/prober.go

```diff
@@ -110,10 +110,10 @@ func (pb *prober) probe(probeType probeType, pod *v1.Pod, status v1.PodStatus, c
 	if err != nil || (result != probe.Success && result != probe.Warning) {
 		// Probe failed in one way or another.
 		if err != nil {
-			klog.V(1).ErrorS(err, "Probe errored", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
+			klog.ErrorS(err, "Probe errored", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
Could you tell me why we should modify this to “Add response body in failed HTTP Probe message”？

Reply:
When checking whether output is print to in logs, I found that the two codes can be optimized. V(1) is unnecessary. Therefore, I changed the code accordingly.

I've reverted this change and re-introduced a new Pull Request to fix the code of klog.V(1).